Hazelnut/src/Panels/SceneHierarchyPanel.cpp

```diff
@@ -323,6 +323,15 @@ namespace Hazel {
 		DrawComponent<SpriteRendererComponent>("Sprite Renderer", entity, [](auto& component)
 		{
 			ImGui::ColorEdit4("Color", glm::value_ptr(component.Color));
+
+			// Texture UVs Offset
+			ImGui::Text("UV Offset");
+			ImGui::SameLine(ImGui::GetContentRegionAvail().x * 0.5f);
+			ImGui::SetNextItemWidth(100.0f);
+			ImGui::DragFloat("##UVOffsetX", &component.TextureUVOffset.x, 0.1f, 0.0f, 0.0f, "%.2f");
+			ImGui::SameLine();
+			ImGui::SetNextItemWidth(100.0f);
+			ImGui::DragFloat("##UVOffsetY", &component.TextureUVOffset.y, 0.1f, 0.0f, 0.0f, "%.2f");
```

Comment:
I suggest the same approach as cherno did, create a separate function, similar to [`DrawVec3Control`](https://github.com/TheCherno/Hazel/blob/d14d33fb9c4c49b049a8a467aab823a1a82ad33f/Hazelnut/src/Panels/SceneHierarchyPanel.cpp#L102-L166):
```cpp
static void DrawVec2Control(const std::string& label, glm::vec2& values, float resetValue = 0.0f, float columnWidth = 100.0f)
```